app/views/layouts/record_not_found.html.erb

```diff
@@ -0,0 +1,23 @@
+<html>
+  <head>
+    <title><%= t('shared.app_name') %></title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', 'sb-admin-2.min','landing-page-style', media: 'all', 'data-turbolinks-track': 'reload' %>
+  </head>
+  <body id="page-top">
```

Comment:
Use IDs only when needed. And follow the convention for ID naming. Verify in the whole PR